### PR TITLE
Add platform type to the triage ticket

### PR DESCRIPTION
**Pull request by mkowalski**

### CODE REVIEW

File: tools/add_triage_signature.py [hunk 1]
```diff
@@ -347,6 +348,7 @@ def build_description(self, url, cluster_md):
                         "logs_url": self._logs_url_to_ui(url).strip('/') + '/',
                         "domain": cluster_md['email_domain'],
                         "openshift_version": cluster_md['openshift_version'],
+                        "platform_type": cluster_md['platform']['type'],
```
Comment (osherdp) [inline tools/add_triage_signature.py:351]: If it's an optional field (judging from code below), then we should probably do:
```suggestion
                        "platform_type": cluster_md.get('platform', {}).get('type', 'N/A'),
```

**Review by osherdp — CHANGES_REQUESTED** [review]

File: tools/add_triage_signature.py (hunk 1 above)
Comment (mkowalski) [reply tools/add_triage_signature.py:351]: It is not, I expect this field to always exist, but anyway a good suggestion

**Review by osherdp — APPROVED** [review]

File: tools/add_triage_signature.py [hunk 2]
```diff
@@ -457,6 +459,12 @@ def extract_cluster_md_from_existing_labels():
             if cluster_md['user_managed_networking'] is True or cluster_md['user_managed_networking'] == 'true':
                 update_fields["labels"].append("FEATURE-User-Managed-Networking")
 
+        # (mko) Platform is not implemented as a separate feature flag, but we want to be able to filter
+        #       it in Jira using labels. For this reason we are extracting it manually from the cluster payload.
+        if 'platform' in cluster_md:
+            if 'type' in cluster_md['platform']:
+                update_fields["labels"].append("FEATURE-Platform-%s", cluster_md['platform']['type'])
```
[the remark this answers is not in the document]
Comment (mkowalski) [reply tools/add_triage_signature.py:466]: Looks like you've lost the race with @osherdp

[the remark this answers is not in the document]
Comment (omertuc) [reply tools/add_triage_signature.py:466]: lucky you

[the remark this answers is not in the document]
Comment (osherdp) [reply tools/add_triage_signature.py:466]: that's why it's good leaving PRs in hold a bit, letting a couple of reviewers to have the chance to comment :joy:

**Pull request merged**